specs/spec.js

```diff
@@ -1,20 +1,25 @@
+/* eslint-disable global-require */
 /* eslint-disable no-undef */
 const { expect } = require('chai');
 const mongoose = require('mongoose');
-const { Price } = require('../database/models/prices');
-const { Seller } = require('../database/models/sellers');
+const { Price, retrievePrices } = require('../database/models/prices');
+const { Seller, retrieveSellers } = require('../database/models/sellers');
 
 describe('Database seeded', () => {
   before(() => {
-    mongoose.connect('mongodb://localhost/seller-catalog', { useMongoClient: true });
+    mongoose.connect('mongodb://localhost/seller-catalog', {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+      useCreateIndex: true,
+    });
   });
```

Comment:
Should these tests be connecting to production db?

Reply:
Setting lower env config not part of this story